fotoapparat/src/main/java/io/fotoapparat/hardware/v1/capabilities/CapabilitiesFactory.java

```diff
@@ -87,4 +91,25 @@ public Capabilities fromParameters(Camera.Parameters parameters) {
         return result;
     }
 
+    private Set<Range<Integer>> extractPreviewFpsRanges(Camera.Parameters parameters) {
```

Comment:
Agreed. 
I started to write test for ```v2.CapatibilitiesFactory``` and faced the problem of mocking ```characteristics.getTargetFpsRanges()``` that returns Android native ```Range``` array, because this class was introduced in Lollipop. I think I should write Instrumented Uni Test in this case, but I don't know how to do it right. Any advise?